src/index.jsx

```diff
@@ -46,6 +46,10 @@ var MaskedInput = React.createClass({
     if (this.props.pattern !== nextProps.pattern) {
       this.mask.setPattern(nextProps.pattern, {value: this.mask.getRawValue()})
     }
+    if (this.props.value !== nextProps.value) {
+      
+      this.mask.setValue(nextProps.value, {value: this.mask.getRawValue()})
```

Comment:
what is the purpose of the second argument here? https://github.com/insin/inputmask-core#setvaluevalue-string